app/models/vote.rb

```diff
@@ -0,0 +1,20 @@
+class Vote < ApplicationRecord
+  #     t.integer :user_id
+  #     t.integer :work_id
+  #     t.date :date
+  #     t.integer :value
+  belongs_to :user
+  belongs_to :work
+  #TODO: ensure value is 1 or -1
+
+  def user
+    User.find(self.user_id)
+  end
```

Comment:
You shouldn't need this method if you have `belongs_to :user` above. Think of it like an `attr_accessor`.